contrib/gitian-build.py

```diff
@@ -11,7 +11,7 @@ def setup():
     if args.kvm:
         programs += ['python-vm-builder', 'qemu-kvm', 'qemu-utils']
     elif args.docker:
-        programs += ['docker.io']
+        programs += ['docker.io' if args.is_bionic else 'docker-ce']
```

Comment:
Sorry for nit-picking this so much, but the package is also named `docker.io` in xenial: https://packages.ubuntu.com/xenial/docker.io

And debian has neither docker.io, nor docker-ce, so I don't really see why this change is here?

Reply:
Add an error message if no docker found